google_auth_oauthlib/flow.py

```diff
@@ -109,7 +109,8 @@ def __init__(
             redirect_uri (str): The OAuth 2.0 redirect URI if known at flow
                 creation time. Otherwise, it will need to be set using
                 :attr:`redirect_uri`.
-
+            code_verifier: random string of 43-128 chars used to verify the
+                key exchange.using PKCE. Auto-generated if not provided.
```

Comment:
Sphinx likely needs the extra blank line between this argument and the `.. _client secrets:` label.